Tasks/Loktionova/Task1/task1.py

```diff
@@ -0,0 +1,5 @@
+sum = 20000
+rate = 0.15
+term = 60
+deposit = sum * (1 + rate / 12) ** term
+print (int (deposit))
```

Comment:
this value should be properly rounded to the second digit after decimal point to display number of 'kopeiki', this is a logical mistake, -15 points